users/hillszach/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Zach-personal
+layout: default
+author: Zach Hills
+---
+# Zachary Hills
+
+<img src="{{ site.url }}users/hillszach/images/37193418_888068498052141_2133433882261848064_o.jpg" height="250" width="250">
+
+Graduate Assistant at Oregon State, Corvallis
+
+M.S. Nuclear Engineering, anticipated 2019
+
+<a href="https://www.linkedin.com/in/zachary-hills-56867873/" target="top"> LinkedIn </a>
+
+Email: hillsz[at]oregonstate[dot]edu
+--------------
+
+## Personal Summary
+
+In the Fall, I am entering my second year of Nuclear Engineering graduate education at Oregon State University under advisor Dr. Todd Palmer. Born in Kansas City and raised in Overland Park, KS, I chose to pursue an undergraduate education in science and technology at the University of Kansas in Lawrence, KS. I began college as a policy debater and learning about carbon driven climate change motivated me to pursue clean energy technology.  My education has explored the air pollution problem and many different research driven ideas for a solution. I have come to understand nuclear technology as most promising for future innovation to cheap, clean energy.
+
+***
+
+## Research Interests
+
+Enhancing our software tools to simulate fuel performance will significantly further efforts to improve expert and public opinions of the market for reactors and ultimately help sell Small Modular Reactors. I am interested in developing the code for particle simulations and multi-physics solvers. 
+
+Currently, I am working in the Reactor Physics group under the advisement of Dr. Mark DeHart at Idaho National Lab. We are developing a I/O linkage program that synchronizes Monte Carlo transport and BISON fuel performance simulations. 
+
+***
+
+## Research Experience
+
+2014: University of Kansas Department of Mathematics, Climate change data modeling
+
+2015-2016: University of Kansas Department of Molecular Biosciences: Enzymology related to siderophore biosythensis
+
+2018: Oregon State University Department of Nuclear Engineering: Reactor physics analysis
+
+***
+
+## Community Involvement
+
+I hope to get many students at Oregon State involved in the discussion of nuclear technology!
```

Comment:
Cool! Maybe add a one liner saying how you'll do it. "...get them involved by..." hosting a Q&A session, etc.